src/utils.js

```diff
@@ -0,0 +1,6 @@
+export const isPasswordEqual = (password: string, passwordConfirm) => password === passwordConfirm;
```

Comment:
create a folder on `src/` named `utils` and place this on something like `formValidations.js`